src/Microsoft.SqlTools.Hosting/Hosting/Protocol/MessageDispatcher.cs

```diff
@@ -134,7 +135,7 @@ public void Stop()
                 requestType.MethodName,
                 async (requestMessage, messageWriter) =>
                 {
-                    Logger.Write(TraceEventType.Verbose, $"Processing message with id[{requestMessage.Id}], of type[{requestMessage.MessageType}] and method[{requestMessage.Method}]");
+                    Logger.Write(TraceEventType.Verbose, $"Processing message with id[{requestMessage.Id}], of type[{requestMessage.MessageType}] and method[{requestMessage.Method}], Content[{requestMessage.Contents.ToString(Formatting.None)}]");
```

Comment:
The issue with this is that message contents can contain PII like passwords and access tokens - and so without proper warning/notification that they may be logging sensitive information to disk I would rather we didn't add this. Or at least make it an opt-in thing where the request handlers can say they want to log the additional content if they're sure it won't be logging anything sensitive.

Reply:
good point, I've removed the request content logging.